### PR TITLE
fixes #70 : fixed the gps tracking app

### LINKED ISSUE #70: Fixing the GPS device tracking app
Presently the GPS device tracking app is just showing a blank white screen without any functionality

### PR DESCRIPTION
**Pull request by djmgit**
Updated the api call in index.html

demo link : 
https://djmgit.github.io/apps.loklak.org/apps/NMEA/

### CODE REVIEW

File: NMEA/index.html [hunk 1]
```diff
@@ -83,7 +83,12 @@ <h4 class="modal-title">Enter the stream URL</h4>
         var cUrl = window.location.href;
         var pName = window.location.pathname;
         var baseUrl = cUrl.split(pName)[0]
-		var urlComplete = baseUrl+'/api/nmea.txt?stream='+url;
+		var urlComplete = "";
+        if (baseUrl === "http://apps.loklak.org") {
+            urlComplete = 'http://api.loklak.org/api/nmea.txt?stream='+url;
+        } else {
+            urlComplete = baseUrl + '/api/nmea.txt?stream='+url;
+        }
```
Comment (sudheesh001) [inline NMEA/index.html]: Looks good here.

File: NMEA/index.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
 	<meta name="author" content="@sudheesh001"> <!-- CHANGE THIS -->
 	<link rel="icon" type="image/png" href="../../artwork/favicon.png">
 	<link href="../../css/bootstrap.min.css" rel="stylesheet">
-	<script src="../../js/jquery.js"></script>
+	<script src="../../js/jquery.min.js"></script>
```
Comment (sudheesh001) [inline NMEA/index.html]: Doesn't look like the right path to me. Is this not throwing a `404` error in the browser console ?

**Comment by djmgit** [comment]
@sudheesh001 The js and css folders are located one level up with respect to NMEA/index.html, so I have changed the paths, although the previous paths were not giving any error in my console!

**Comment by mariobehling** [comment]
Please resolve conflicts. Thanks.

**Comment by mariobehling** [comment]
@djmgit This PR is open for a very long time now. Please resolve the conflicts or close it.

**Comment by djmgit** [comment]
@mariobehling I will be submitting a new PR for this issue, therefore closing this for now

**Pull request closed without merging**